### ISSUE TITLE
Android14 Beta4 crop not working (Pixel 6 Pro)

### ISSUE THREAD

**Issue by dhwz** [Genymobile/scrcpy#4162]
- [X] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [X] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Windows 11
 - scrcpy version: 2.1
 - installation method: Win x64 from GitHub
 - device model: Pixel 6 Pro
 - Android version: 14 Beta 4

**Describe the bug**
Tried to crop the black bar from the screen which is always visible on Pixel 6 Pro in Landscape mode. It worked perfectly on Android 13
but crop is completely ignored on Android 14 Beta 4

--crop 1440:2600:0:120 removed the black bar on Android 13 but doesn't do anything on Android 14
Note: Mouse position is still wrong if the crop is applied and also the scrcpy output says it has cropped
INFO: Texture: 1440x2480 vs. INFO: Texture: 1440x2600  if crop is 0

```
scrcpy.exe -w --video-bit-rate 25M --video-codec=h265 --audio-bit-rate=256K --audio-codec=opus --crop 1440:2600:0:120
scrcpy 2.1 <https://github.com/Genymobile/scrcpy>
C:\STORAGE\scrcpy-win64-v2.0\scrcpy-server: 1 file pushed, 0 skipped. 77.6 MB/s (56955 bytes in 0.001s)
[server] INFO: Device: [Google] google Pixel 6 Pro (Android 14)
INFO: Renderer: direct3d
INFO: Texture: 2480x1440
INFO: Texture: 1440x2480
[server] INFO: Device rotation requested: landscape
INFO: Texture: 2480x1440
```

**Comment by dhwz** [comment]
@rom1v installed Beta 5 now, seems it's still broken. Do you have any idea what has changed from Android 13?

**Comment by rom1v** [comment]
Nope, I don't know, they changed things related to display, which also broke `--turn-screen-off` (#3784). To be investigated (once the source code of Android 14 is released)…

**Comment by SharkWipf** [comment]
FWIW: seems like the source is released now: https://android-developers.googleblog.com/2023/10/android-14-is-live-in-aosp.html

I can also confirm crop not working\* on the Pixel 8 Pro, so nothing's changed there since beta it seems.

\* The inputs do get "cropped" though, so any touches go to wrong places.

**Comment by rom1v** [comment]
See https://github.com/Genymobile/scrcpy/issues/4011

**Comment by dhwz** [comment]
@rom1v Thank you very much, can you provide updated binaries so I can test it?

**Comment by rom1v** [comment]
Please test a new version in #5455, a binary is provided.

**Comment by dhwz** [comment]
Thanks again, verified it's working as expected.
